main_test.go

```diff
@@ -229,30 +206,56 @@ func waitForServer() bool {
 		time.Sleep(50 * time.Millisecond)
 	}
 
-	if !worked {
-		failMainTest("timeout on server starting: " + lastClientErr.Error() + "\n" + string(lastClientOutput))
+	So(worked, ShouldEqual, true)
+}
+
+func (s *TestServer) Shutdown() error {
+	err := s.cmd.Process.Signal(os.Interrupt)
+
+	errCh := make(chan error, 1)
+
+	go func() { errCh <- s.cmd.Wait() }()
+
+	select {
+	case errb := <-errCh:
+		return errors.Join(err, errb)
+	case <-time.After(5 * time.Second):
+		return errors.Join(err, s.cmd.Process.Kill())
+	}
+}
+
+func remoteDBBackupPath() string {
+	collection := os.Getenv("IBACKUP_TEST_COLLECTION")
+	if collection == "" {
+		return ""
 	}
 
-	return worked
+	return filepath.Join(collection, "db.bk")
 }
 
 func TestStatus(t *testing.T) {
-	SkipConvey("With no server, status fails", t, func() {
-		confirmOutput(t, []string{"status"}, 1, "you must supply --url")
-	})
-
-	Convey("With no sets defined, status returns no sets", t, func() {
-		confirmOutput(t, []string{"status"}, 0, `Global put queue status: 0 queued; 0 reserved to be worked on; 0 failed
+	Convey("With a started server", t, func() {
+		s := NewTestServer(t)
+		So(s, ShouldNotBeNil)
+
+		/*
```

Comment:
We shouldn't have commented out code. Why is the Convey skipped?

Reply:
It was commented out because it needed some rewriting (very little rewriting as it happens) to work, and I missed it when committing.

I've now pulled it out as its own test (TestNoServer).